andela/ah-commando-backend
# #167190440 Attach authors to articles created by them

#### What does this PR do?
> - Attach authors to each article created by them
> - Prevent authors from editing and deleting articles that do not belong to them
#### Description of Task to be completed?
> - refactor the create articles function to attach the author that created an article to the article
> - authors can only edit or delete articles created by them
#### How should this be manually tested?
> - clone the repo using the command `git clone [repo url]`
> - run `npm install` to install all dependencies needed for the code to run
> - navigate to the URL `localhost:5040/articles` and make a `POST` to create a new article
> - Add a check to ensure that the only the owner of an article can either edit or delete the article
#### Any background context you want to provide?
> - none
#### What are the relevant pivotal tracker stories?
[#167190440](https://www.pivotaltracker.com/story/show/167190440)
#### Screenshots (if appropriate)

<img width="985" alt="Screenshot 2019-08-05 at 10 20 21 AM" src="https://user-images.githubusercontent.com/35861980/62453668-b1dd5680-b76a-11e9-99ac-46fb5188f8c3.png">

#### Questions:
> - none
